cdap-ui/app/cdap/components/PreviewData/index.tsx

```diff
@@ -91,9 +91,11 @@ const PreviewDataViewBase: React.FC<IPreviewDataViewProps> = ({
     off: { value: PreviewMode.Table, label: PreviewMode.Table },
   };
 
+  let shouldShowRecordView;
```

Comment:
nit: should we default this to false just to maintain the variable as a boolean?